addons/cookoff/functions/fnc_cookOffBox.sqf

```diff
@@ -44,9 +44,9 @@ if (local _vehicle) then {
 
         // These functions are smart and do all the cooking off work
         if (local _box) then {
-            _box call FUNC(secondaryExplosions);
             if (_box getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
```

Comment:
Almost feel like this setting dependency should either be removed for boxes, or the `GVAR(enableAmmobox)` setting homogenised with this one (see XEH_postInit.sqf line 74).